browser/brave_vpn/win/brave_vpn_wireguard_service/service/tunnel_utils.cc

```diff
@@ -0,0 +1,63 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/brave_vpn/win/brave_vpn_wireguard_service/service/tunnel_utils.h"
+
+#include "base/logging.h"
+#include "base/win/registry.h"
+#include "brave/components/brave_vpn/common/wireguard/win/service_constants.h"
+#include "brave/components/brave_vpn/common/wireguard/win/storage_utils.h"
+
+namespace brave_vpn {
+
+namespace wireguard {
+
+namespace {
+constexpr wchar_t kBraveWireguardConfigKeyName[] = L"ConfigPath";
+}  // namespace
+
+// Increments number of usages for the wireguard tunnel service.
+void IncrementWireguardUsageFlag() {
```

Comment:
nit: to align with `ResetWireguardTunnelUsageFlag()` method name.
```suggestion
void IncrementWireguardTunnelUsageFlag() {
```

Reply:
done